.gitlab-ci.yml

```diff
@@ -49,10 +49,12 @@ include:
     - ci/install-and-build.yml
     - ci/prebuild.yml
     - ci/packages/components.yml
-    - ci/packages/suite-misc.yml
     - ci/packages/suite-web.yml
     - ci/packages/suite-desktop.yml
 #    - ci/packages/suite-native.yml
     - ci/packages/rollout.yml
     - ci/packages/landing-page.yml
     - ci/packages/suite-web-landing.yml
+    # to be run only on scheduled pipelines
+    - ci/schedules.yml
+    
```

Comment:
Your editor sucks :). There is an extra `\t`.

Reply:
Sorry for the nit-picks :angel:

Reply:
Won't fix as long as there is not automatic linting tool for this in place :rebelion:

Reply:
:facepalm: Okay then, what can I do :lab_coat:.